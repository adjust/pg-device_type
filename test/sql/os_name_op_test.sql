BEGIN;
CREATE EXTENSION device_type;
SELECT 'phone'::device_type > 'ipod'::device_type;
SELECT 'phone'::device_type >= 'ipod'::device_type;
SELECT 'ipod'::device_type < 'pc'::device_type;
SELECT NOT('phone'::device_type < 'ipod'::device_type);
SELECT NOT('phone'::device_type <= 'ipod'::device_type);
ROLLBACK;
BEGIN;
CREATE EXTENSION device_type;
SELECT 'phone'::device_type > 'ipod'::device_type;
SELECT 'phone'::device_type >= 'ipod'::device_type;
SELECT NOT('phone'::device_type < 'ipod'::device_type);
SELECT NOT('phone'::device_type <= 'ipod'::device_type);
ROLLBACK;
BEGIN;
CREATE EXTENSION device_type;
SELECT NOT('phone'::device_type = 'ipod'::device_type);
SELECT 'ipod'::device_type = 'ipod'::device_type;
ROLLBACK;
BEGIN;
CREATE EXTENSION device_type;
SELECT 'phone'::device_type != 'ipod'::device_type;
SELECT NOT('ipod'::device_type != 'ipod'::device_type);
ROLLBACK;
