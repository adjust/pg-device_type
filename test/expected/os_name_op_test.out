BEGIN;
CREATE EXTENSION device_type;
SELECT 'phone'::device_type > 'ipod'::device_type;
 ?column? 
----------
 t
(1 row)

SELECT 'phone'::device_type >= 'ipod'::device_type;
 ?column? 
----------
 t
(1 row)

SELECT 'ipod'::device_type < 'pc'::device_type;
 ?column? 
----------
 t
(1 row)

SELECT NOT('phone'::device_type < 'ipod'::device_type);
 ?column? 
----------
 t
(1 row)

SELECT NOT('phone'::device_type <= 'ipod'::device_type);
 ?column? 
----------
 t
(1 row)

ROLLBACK;
BEGIN;
CREATE EXTENSION device_type;
SELECT 'phone'::device_type > 'ipod'::device_type;
 ?column? 
----------
 t
(1 row)

SELECT 'phone'::device_type >= 'ipod'::device_type;
 ?column? 
----------
 t
(1 row)

SELECT NOT('phone'::device_type < 'ipod'::device_type);
 ?column? 
----------
 t
(1 row)

SELECT NOT('phone'::device_type <= 'ipod'::device_type);
 ?column? 
----------
 t
(1 row)

ROLLBACK;
BEGIN;
CREATE EXTENSION device_type;
SELECT NOT('phone'::device_type = 'ipod'::device_type);
 ?column? 
----------
 t
(1 row)

SELECT 'ipod'::device_type = 'ipod'::device_type;
 ?column? 
----------
 t
(1 row)

ROLLBACK;
BEGIN;
CREATE EXTENSION device_type;
SELECT 'phone'::device_type != 'ipod'::device_type;
 ?column? 
----------
 t
(1 row)

SELECT NOT('ipod'::device_type != 'ipod'::device_type);
 ?column? 
----------
 t
(1 row)

ROLLBACK;
